python/tvm/relax/training/optimizer.py

```diff
@@ -333,38 +335,38 @@ def get_function(self) -> Function:
         dtype = self._dtype
 
         # input variables
-        param_var = Var("params", rx.TupleStructInfo([p.struct_info for p in plist]))
-        grad_var = Var("gradients", rx.TupleStructInfo([p.struct_info for p in plist]))
-        state_var = Var("optim_states", rx.TupleStructInfo([rx.TensorStructInfo((), "int64")]))
+        param_var = Var("params", TupleStructInfo([p.struct_info for p in plist]))
+        grad_var = Var("gradients", TupleStructInfo([p.struct_info for p in plist]))
+        state_var = Var("optim_states", TupleStructInfo([TensorStructInfo((), "int64")]))
 
         # constants
-        lr = rx.const(self.lr, dtype)
```

Comment:
Not a big problem. Personally prefer `rx.xxx` since it is clear that it's a Relax function, while `abs` and `const` may be a bit ambiguous. No need to change if you persist in a short name :)

Reply:
Sometimes `rx.xxx` under the `relax` package will cause circular import so we change them to relative import.